package.json

```diff
@@ -6,29 +6,33 @@
         "node": "^18.18.0 || ^20.9.0 || >=21.1.0"
     },
     "main": "lib/index.js",
+    "types": "types/index.d.ts",
     "files": [
         "lib/",
-        "configs/"
+        "configs/",
+        "types/index.d.ts"
     ],
     "peerDependencies": {
         "eslint": "^8.23.0 || >=9.0.0-0"
     },
     "dependencies": {
         "@eslint-community/eslint-utils": "^4.4.0",
+        "@types/eslint": "^8.56.2",
         "enhanced-resolve": "^5.15.0",
         "eslint-plugin-es-x": "^7.5.0",
         "get-tsconfig": "^4.7.0",
         "globals": "^14.0.0",
         "ignore": "^5.2.4",
         "is-builtin-module": "^3.2.1",
-        "is-core-module": "^2.12.1",
```

Comment:
is this an expected change?

Reply:
This is expected as I replaced all references to `is-core-module` with `is-builtin-module`, I figured it's best to have just one module list.

![image](https://github.com/eslint-community/eslint-plugin-n/assets/2230835/95d60827-9481-4d54-8dba-88857336d529)

Especially as we just did this everywhere:
```js
// Note: "999" arbitrary to check current/future Node.js version
.filter(([name]) => isCoreModule(name, "999"))
```